common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -176,6 +198,18 @@
                 this.updatePlayTime(time);
             },
 
+            handleCaptionToggle: function(event) {
+                var KEY = $.ui.keyCode,
+                    keyCode = event.keyCode;
+
+                switch(keyCode) {
+                    // KEY.ENTER works inherently so we don't need to script it explicitly
```

Comment:
With or without a screen reader or any JavaScript, pressing Enter would activate (press, click, what have you) the control. By default, pressing the Spacebar scrolls the page, so we provide scripting for it to behave like the Enter key.